utils/check_template_test_coverage.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+from glob import glob
+import os.path
+import re
+import sys
+
+import iniparse
```

Comment:
`import configparser` from core python works. Iniparse is very handy for writing, which we do not do. (And some other things.)

Reply:
sigh... `iniparse` is very much nicer to work with in general too...